### PR TITLE
Adding credits to front end

### PR DESCRIPTION
**Pull request by gizzon**
**Issue**

This PR doesn't close an issue per se, but rather builds upon a recent PR that added credits to the database. This PR utilizes those entries and displays them on the website. These changes can be seen:

1) As a credit counter under the CRNs list
2) On the course info popup while on schedule
3) On the course page

**Database Changes/Migrations**

No database changes

**Photos**
Messages Image(1113238466)
Messages Image(1552780918)
Messages Image(2021742158)
Messages Image(3966734186)

### CODE REVIEW

**Comment by marchdan** [comment]
@jshom Do you know why there is no temp deploy on here?

**Comment by jshom** [comment]
@marchdan not sure, maybe we need a merge/rebase from master

File: src/web/src/pages/CourseScheduler.vue [hunk 1]
```diff
@@ -107,6 +108,16 @@
         <br />
         <br />
       </span>
+      <span v-if= "courseInfoModalCourse.min_credits == courseInfoModalCourse.max_credits">
+        Credits: {{ courseInfoModalCourse.min_credits }}
+        <br />
+        <br />
+      </span>
+      <span v-if= "courseInfoModalCourse.min_credits != courseInfoModalCourse.max_credits">
```
Comment (IkeLyons) [inline src/web/src/pages/CourseScheduler.vue:116]: Think this could just be a v-else

File: src/web/src/pages/CoursePage.vue [hunk 2]
```diff
@@ -75,6 +76,15 @@ export default {
       treeData.name = this.courseName;
       return treeData;
     },
+    getCredits() {
+      var credits;
+      if (this.courseObj.min_credits != this.courseObj.min_credits){
```
Comment (IkeLyons) [inline src/web/src/pages/CoursePage.vue:81]: I think this is a typo, left hand side should probably be `this.courseObj.max_credits`?

**Review by IkeLyons — CHANGES_REQUESTED** [review]
Looks good! just a couple small changes

**Comment by RichtXO** [comment]
With all the issues away, this looks nice! Let's try to get the credits displayed while the user is exploring the courses in the explore tab.

**Review by IkeLyons — APPROVED** [review]

**Comment by marchdan** [comment]
> With all the issues away, this looks nice! Let's try to get the credits displayed while the user is exploring the courses in the explore tab.

@RichtXO Is this something that still needs to be added to this PR?

**Comment by RichtXO** [comment]
> > With all the issues away, this looks nice! Let's try to get the credits displayed while the user is exploring the courses in the explore tab.
> 
> @RichtXO Is this something that still needs to be added to this PR?

I guess we can make another issue for it, but I think it's best to add them in at the same time.

**Comment by RichtXO** [comment]
Going more in details, I see that there's a screenshot having the credits in the explore page, but for some reason, I don't see it on my end.

**Comment by gizzon** [comment]
> Going more in details, I see that there's a screenshot having the credits in the explore page, but for some reason, I don't see it on my end.

Thanks for making those changes Rich. I was having trouble pushing my changes. I'm not sure why you're having that issue. I'll check it out now.

File: src/web/src/pages/CoursePage.vue [hunk 3]
```diff
@@ -49,7 +54,7 @@ export default {
       courseObj: {},
       selectedSemester: String,
       backRoute: String,
-      breadcrumbNav: [
+      breadcrumbNav: [
```
Comment (jshom) [inline src/web/src/pages/CoursePage.vue:57]: Please remove spare space at the end of the line

File: src/web/src/pages/CourseScheduler.vue [hunk 4]
```diff
@@ -452,6 +461,17 @@ export default {
...
+     * Returns sum of credits being taken from all selected sections
+     */
+    totalCredits() {
+      var array = Object.values(this.selectedCourses).map((c) => c.max_credits);
+    
+      // Getting sum of numbers
+      var sum = array.reduce(function(a, b){
+        return a + b;}, 0);
+      return sum;
+    },
```
Comment (jshom) [inline src/web/src/pages/CourseScheduler.vue:474]: hi proglang

**Review by jshom — APPROVED** [review]

**Pull request merged**